mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/Profile.java

```diff
@@ -2,71 +2,29 @@
 
 import java.time.LocalDate;
 
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
 import com.google.gson.annotations.SerializedName;
+import com.mx.path.core.common.model.Internal;
 import com.mx.path.model.mdx.model.MdxBase;
 
+@Data
+@EqualsAndHashCode(callSuper = true)
 public final class Profile extends MdxBase<Profile> {
   @SerializedName("birth_date_on")
   private LocalDate birthDate;
-  @SerializedName("first_name")
   private String firstName;
   private Gender gender;
-  @SerializedName("last_name")
   private String lastName;
-  @SerializedName("middle_name")
   private String middleName;
   private String ssn;
 
-  public Profile() {
-  }
-
-  public LocalDate getBirthDate() {
-    return birthDate;
-  }
-
-  public void setBirthDate(LocalDate birthDate) {
-    this.birthDate = birthDate;
-  }
-
-  public String getFirstName() {
-    return firstName;
-  }
-
-  public void setFirstName(String firstName) {
-    this.firstName = firstName;
-  }
-
-  public Gender getGender() {
-    return gender;
-  }
-
-  public void setGender(Gender gender) {
-    this.gender = gender;
-  }
-
-  public String getLastName() {
-    return lastName;
-  }
-
-  public void setLastName(String lastName) {
-    this.lastName = lastName;
-  }
-
-  public String getMiddleName() {
-    return middleName;
-  }
-
-  public void setMiddleName(String middleName) {
-    this.middleName = middleName;
-  }
-
-  public String getSsn() {
-    return ssn;
-  }
-
-  public void setSsn(String ssn) {
-    this.ssn = ssn;
-  }
+  // internal fields
+  @Internal
+  private Boolean remoteDepositEligible;
+  @Internal
+  private Boolean remoteDepositEnrolled;
```

Comment:
I'm wondering if these should be `Boolean` or `boolean`. They are `Boolean` in `RemoteUser`, but we use `boolean` for the [Account internal fields](https://github.com/mxenabled/path-mdx-model/blob/603d682ddea2b46e53d00745d12c0463e0bbdca8/mdx-models/src/main/java/com/mx/path/model/mdx/model/account/Account.java#L135-L146). I guess I don't have a strong preference either way. 🤔 

I also like the comment block in [Account](https://github.com/mxenabled/path-mdx-model/blob/603d682ddea2b46e53d00745d12c0463e0bbdca8/mdx-models/src/main/java/com/mx/path/model/mdx/model/account/Account.java#L118-L122) explaining how internal fields work. I think it would be helpful to copy that over to this class.

Reply:
I was also torn on that... We seem to have a mix of Boolean and boolean throughout the models. I ended up copying RemoteUser. I'll get some other opinions and add that comment

Reply:
I think I prefer `boolean` generally because it's cleaner to work with, since you don't need to handle the null case. Also, the lombok getter is a bit more readable.

`Boolean` usage:
```java
user.getRemoteDepositEligible() == null || !user.getRemoteDepositEligible()
```

`boolean` usage:
```java
!user.isRemoteDepositEligible()
```

Reply:
Makes sense to me! Fixed